games/a.yaml

```diff
@@ -51,6 +51,31 @@
   updated: 2016-12-11
   url: https://achtungkurve.com/
 
+- name: Achtung, die Kurve!
+  images:
+  - https://kurve.se/screenshots/kurve-round.png
+  - https://kurve.se/screenshots/kurve-splash.png
+  - https://kurve.se/screenshots/kurve-lobby.png
+  info: The original <kbd>ZATACKA.EXE</kbd> in the browser
+  framework:
+  - HTML
+  - CSS
```

Comment:
these aren't really frameworks; this should be removed

Reply:
Agree. Only reason I included those was because I copied the existing _Achtung, die Kurve!_ entry (by maechler) and modified it.

Reply:
Should I make this change, `git commit --amend` and then `git push --force` (to the `kurve` branch of my fork)?

Reply:
@SimonAlling Actually I did not add the game myself initially, but I just checked the initial commit when my implementation was added, there it said: `lang: [Javascript, CSS, HTML]`
I do not why this was moved to `framework`.